### PR TITLE
SPLICE-1138 Null Fraction Computational Error

### PR DESCRIPTION
**Pull request by jleach4**
Careless mistake when computing the percentage.  Added test for coverage.

### CODE REVIEW

File: splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsFunctions.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public static float STATS_NULL_FRACTION(ColumnStatisticsImpl itemStatistics){
         if(itemStatistics==null) return 0;
         if (itemStatistics.notNullCount() == 0)
             return 0;
```
Comment (genedavis) [inline splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsFunctions.java]: Should return 1

File: splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsFunctions.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public static float STATS_NULL_FRACTION(ColumnStatisticsImpl itemStatistics){
...
-        return itemStatistics.nullCount()/itemStatistics.notNullCount();
+        return ((float) itemStatistics.nullCount()/ (float) itemStatistics.notNullCount());
```
Comment (jleach4) [inline splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsFunctions.java]: Nice catch.  Will fix and re-push.  Thanks Gene.

Comment (jleach4) [inline splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsFunctions.java]: Fixed...

**Comment by genedavis** [comment]
+1

**Pull request merged**